cloud-platform/app-deploy.md

```diff
@@ -0,0 +1,205 @@
+
+Now we need to tag the image with the tag provided by ECR, so I can be pushed into the correct repository.
+
+View the **Push Commands** again, and modify the fourth command provided, ensuring the first tag is the one currently used on your machine:
+
+`docker tag cloud-platform-demo-app:latest 926803513772.dkr.ecr.us-west-1.amazonaws.com/cloud-platform-demo-app:latest`
+
+Finish by running the fifth command provided, to push the image to your repository.
+
+`docker push 926803513772.dkr.ecr.us-west-1.amazonaws.com/cloud-platform-demo-app:latest`
+
+A quick refresh of ECR, and your image should now be displayed.
+
+## Configuring deployment files
+
+To deploy an application to the Cloud Platform, a number of deployment files must first be configured.
+
+These deployment files will make reference to your application's Docker image and handle configurations, such as ports and host resolving.
+
+This guide will analyse the deployment files for `Cloud-Platform-Demo-App`.
+
+The deployment files can be used as a basic template for your application, and can be found at:
+
+[https://github.com/ministryofjustice/cloud-platform-demo-app](https://github.com/ministryofjustice/cloud-platform-demo-app)
+
+### Deployment
+
+Deployment files are used to specify core information about an application that is being deployed to the Cloud Platform.
+
+See the contents of the `deployment.yml` file below:
+
+```Yaml
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: cp-demo-app
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: cp-demo-app
+    spec:
+      containers:
+      - name: cp-demo-app
+        image: 926803513772.dkr.ecr.us-west-1.amazonaws.com/cloud-platform-demo-app:latest
+        ports:
+        - containerPort: 80
+```
+
+If you are using the file as a template for your own application, replace the `cp-demo-app` tags with ones suited to your application.
+
+For the value of the `image:` key, you will see the Repository URI, which has been provided by ECR.
+
+If you are using the file as a template for your own application, replace the value of `image:` with the Repository URI of your application, found in ECR.
+
+### Service
+
+Service files are used to specify port and protocol information for your application.
+
+See the contents of the `service.yml` file below:
+
+```Yaml
+kind: Service
+apiVersion: v1
+metadata:
+  name: cp-app-demo-svc
+  labels:
+    app: cp-app-demo-svc
+spec:
+  ports:
+  - port: 80
+    name: http
+    targetPort: 80
+  selector:
+    app: cp-demo-app
+```
+
+If you are using the file as a template for your own application, replace the `cp-demo-app-svc` tags with ones suited to your application.
+
+Also, ensure the `selector:` app tag is the same as specified in the `deployment.yml`.
+
+### Ingress
+
+Ingress files are to use to specify host resolving information for your application.
```

Comment:
I think this should read "[...] to define external access to the application". Technically host resolution would be DNS.